tools/Nuget/template.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Anyone it's ok for me, but when I checked other repos and FAKE documentation that's how it was done: http://fsharp.github.io/FAKE/create-nuget-package.html